Makefile

```diff
@@ -261,10 +261,12 @@ clean: clean-certs clean-containers clean-slice ## Stops all containers and remo
 	$(RM) *.box
 
 test: ## executes the test script on a master
+	# This command is broken, as per
```

Comment:
There were two issues:
* This didn't actually call the script as it `cd`d to the wrong place (perhaps the script moved)
* The script itself is broken

This PR fixes the former but not the latter.